src/Microsoft.Extensions.Caching.Abstractions/IMemoryCache.cs

```diff
@@ -30,5 +30,11 @@ public interface IMemoryCache : IDisposable
         /// </summary>
         /// <param name="key">An object identifying the entry.</param>
         void Remove(object key);
+
+        /// <summary>
+        /// Attempt to remove the given percentage of the total entries in the cache.
+        /// </summary>
+        /// <param name="percentage">The percentage of cache entries to remove.</param>
+        void Compact(double percentage);
```

Comment:
Can you update the doc comment in `MemoryCache` itself to be a proper doc comment? Right now there are some odd `///` comments there but not an actual doc comment. https://github.com/aspnet/Caching/blob/cd57aea9755dc8cbf0dbfa1cfa35babfd78f49d2/src/Microsoft.Extensions.Caching.Memory/MemoryCache.cs#L275-L281